spec.md

```diff
@@ -1577,6 +1658,21 @@ message ControllerServiceCapability {
       // Indicates the SP supports the
       // ListVolumesResponse.entry.published_nodes field
       LIST_VOLUMES_PUBLISHED_NODES = 10;
+
+      // Indicates the SP supports the
+      // ListVolumesResponse.entry.volume_condition field
+      LIST_VOLUMES_VOLUME_CONDITION = 11 [(alpha_enum_value) = true];
+
+      // Indicates the SP supports the GetVolume RPC
+      GET_VOLUME = 12 [(alpha_enum_value) = true];
+
+      // Indicates the SP supports the
+      // GetVolumeResponse.volume_condition field
+      GET_VOLUME_VOLUME_CONDITION = 13 [(alpha_enum_value) = true];
```

Comment:
Instead of both `LIST_VOLUMES_VOLUME_CONDITION` and `GET_VOLUME_VOLUME_CONDITION` let's just have one -- `VOLUME_CONDITION`. Caller can always check for `LIST_VOLUMES` and `GET_VOLUME` capability.